tldw_Server_API/app/core/TTS/adapters/omnivoice_adapter.py

```
@@ -0,0 +1,343 @@
+                    headers=headers,
+                )
+
+            if response.status_code != 200:
+                raise TTSGenerationError(
+                    "OmniVoice sidecar returned an error",
+                    provider=self.PROVIDER_KEY,
+                    details={
+                        "status_code": response.status_code,
+                        "response_text": response.text,
+                    },
+                )
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)